frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -1,20 +1,24 @@
 // TODO: merge with metabase/dashboard/containers/Dashboard.jsx
-/* eslint-disable react/prop-types */
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { Box } from "grid-styled";
-import { t } from "ttag";
 import _ from "underscore";
-import cx from "classnames";
 
-import DashboardHeader from "./DashboardHeader";
-import DashboardGrid from "./DashboardGrid";
-import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
-import Parameters from "metabase/parameters/components/Parameters/Parameters";
-import EmptyState from "metabase/components/EmptyState";
-import { DashboardSidebars } from "./DashboardSidebars";
-
-import DashboardControls from "../hoc/DashboardControls";
+import { FullWidthContainer } from "metabase/styled-components/layout/FullWidthContainer";
+import {
+  DashboardStyled,
+  DashboardLoadingAndErrorWrapper,
+  DashboardBody,
+  HeaderContainer,
+  ParametersAndCardsContainer,
+  ParametersWidgetContainer,
+} from "./Dashboard.styled";
+import DashboardHeader from "../DashboardHeader";
+import DashboardGrid from "../DashboardGrid";
+import ParametersWidget from "./ParametersWidget/ParametersWidget";
+import DashboardEmptyState from "./DashboardEmptyState/DashboardEmptyState";
+import { DashboardSidebars } from "../DashboardSidebars";
+
+import DashboardControls from "../../hoc/DashboardControls";
```

Comment:
Should we rearrange imports here a bit? Usually the closer the imported file is to the project root, the higher it is in the imports list. So e.g. `DashboardControls` should to under the `FullWidthContainer` I guess, imports from the same directory have to be in the bottom, etc.